### PR TITLE
merge Dev

### PR DESCRIPTION
**Pull request by morrning**
Dev branch base errors was solved! system is unstable

### CODE REVIEW

**Comment by meysam1366** [comment]
سلام آقا بابک

خوبی؟شرمنده خبری ازم نیست درگیر یک پروژه دروپال هستم انشاالله تموم که بشه
در خدمتم

بازم ببخشید

**Comment by morrning** [comment]
سلام 
خواهش میکنم مشکلی نیست

File: README.md [hunk 1]
```diff
@@ -1,21 +1,13 @@
 sarkesh
 By Negareh Sarkesh LTD (http://sarkesh.org)
 =======
-Sarkesh is php framework with server side events support for html controls.
+Sarkesh is php content mangment framework with server side events support for html controls.
 php events in sarkesh do with jquery. 
 
 base of sarkesh is on plugins that designed with MVC.
```
Comment (MSalehi) [inline README.md]: Sarkesh is based on plugins which is designed using MVC Architecture.

**Comment by morrning** [comment]
yes that's right.
but  now we're in next version and this file changed about 6 week ago.

**Pull request merged**